assets/prefabs/recipe/cook/RecipeToast.prefab

```diff
@@ -0,0 +1,19 @@
+{
+    "ProcessDefinition": {
+        "processType": "Cooking:Cooking"
+    },
+    "CraftingStationRecipe": {
+        "recipeId": "Hot food|Cooking:Toast",
+
+        "recipeComponents": ["1*Bread"],

```

Comment:
Yea I tried give toast and to craft it as well, I thought it was just something I was doing wrong so I was gonna ask if it was working for you, any ideas on how to fix it?